README.md

```diff
@@ -128,11 +128,28 @@ The `actual` directory always points to the configured screenshot directory.
 
 **Configure snapshot generation**
 
-You can control if diff images are generated for successful tests by setting the following environment variable:
+In order to control the creation of diff images you may want to use the following environment variables which are 
+typically set in field `env` in `cypress.json`. 
```

Comment:
Let's update these to Cypress >= v10

Reply:
AFAIK, it should be enough to replace `cypress.json` by `cypress.config.json`, which i did in both cases